.changeset/witty-rocks-search.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/slider": patch
```

Comment:
Sorry! 😢 The changeset is not needed for this PR, as it is related to refactoring. Please delete this file. @EGASHIRAAkihide